### ISSUE TITLE
Add better documentation to load.loader.loader() (wrapper of load.images.MapImages.loader())

### ISSUE THREAD

**Issue by rwood-97** [Living-with-machines/MapReader#167]
**Is your feature request related to a problem? Please describe.**
Need this as its the main way people interface with the loader.

**Describe the solution you'd like**
Add doc string to give more params. Potentially just pull from load.images..MapImages.loader()

**Describe alternatives you've considered**
Actually give it all the same params as images loader.

**Comment by kallewesterling** [comment]
I didn't touch the `load` module because I think we had been discussing that it needed a rewrite before the docs could be touched. I'd be happy to take a stab at this, if you'd like!

**Comment by rwood-97** [comment]
Yes, I think as soon as PR #163 and PR #160 are merged we should add this in. But you are right that waiting till all the code updates are done is probably a good idea.
